recipes/1-Insights/conditioned-insights.ipynb

```diff
@@ -0,0 +1,309 @@
+    "\n",
+    "For more information on the Global statistics, see the [Global vs Local](https://docs.howso.com/user_guide/concepts/global_vs_local.html) documentation.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "accuracy          0.829000\n",
+       "mcc               0.530553\n",
+       "precision         0.737685\n",
+       "recall            0.796072\n",
+       "mae               0.239578\n",
+       "spearman_coeff         NaN\n",
+       "rmse                   NaN\n",
+       "r2                     NaN\n",
+       "Name: target, dtype: float64"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "global_stats = t.get_prediction_stats()['target']\n",
+    "global_stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Local Prediction Stats\n",
+    "\n",
+    "In addition to global prediction stats, Howso has the unique ability to provide local prediction stats. These prediction stats are more finely tuned to the exact cases in which you are interested in. This can be extremely valuable for workflows such as data exploration and it provides more nuanced performance metrics. The more variance there is in the performance of dataset from region to region, the more powerful this ability is."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.7"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results = t.react(\n",
+    "    test_case,\n",
+    "    context_features=context_features,\n",
+    "    action_features=action_features,\n",
+    "    details = {\n",
+    "        \"prediction_stats\": True\n",
+    "    }\n",
+    ")\n",
+    "\n",
+    "local_stats = results['details']['prediction_stats'][0]\n",
+    "local_stats['accuracy'][action_features[0]]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see how the local prediction stats differ from the global stats. Using local metrics, we can drill into specific cases. For example, in our test case which is manually set to `female`, we can get insights into how well the Trainee fits similiar cases. In the local space, this does not gurantee all of the other cases in the local space are also female, however it increases the chances that they are.\n",
+    "\n",
+    "If we see that the local accuracy drops off a lot from the global accuracy, this can demonstrate that in this region of the data, the Trainee and data do not have the same predictive performance as the rest of the Trainee and data."
```

Comment:
Can we change this sentence to: "If we see the local accuracy is significantly lower than the global accuracy, the Trainee does not have the same predictive power within this region of the data, compared to it's overall performance. This could indicate a need for new and/or different training data to improve the Trainee's performance in this region."